apps/screenshot-worker/wrangler.toml

```diff
@@ -14,6 +14,8 @@ browser = { binding = "MYBROWSER" }
 [[r2_buckets]]
 binding = "MY_BUCKET"
 bucket_name = "incident-screenshot"
+preview_bucket_name = "preview_bucket"
```

Comment:
How comes adding that line?

Reply:
Sorry, I got rid of it. When I first ran the project I got some errors so I thought I needed it to run the project locally.